src/containers/projects/Projects.js

```diff
@@ -78,6 +86,9 @@ export default function Projects() {
         </div>
         <Button text={"More Projects"} className="project-button" href={socialMediaLinks.github} newTab={true} />
       </div>
-    </Fade>
```

Comment:
Does it work if we put `Fade` inside Suspense, if yes then let's keep it, in earlier files too.

Reply:
Well adding `Fade` doesn't show the loading screen, so just leaving it as is.